client/src/components/AssignmentBoard/ProjectBox.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { Box, Typography } from '@mui/material';
+import VolunteerBox from './VolunteerBox';
+
+export default function ProjectBoard({ volunteers, project, classes }) {
+    const [isActive, setIsActive] = useState(false);
+
+    return (
+        <Box 
+            className={classes.projectCard} 
+            mt="8px" 
+            mb="16px">
+
+            <Typography variant="h6" mb="16px">{project.name}</Typography>
```

Comment:
Looks like long names are overflowing out of the box: 
<img width="803" alt="Screen Shot 2022-04-20 at 11 47 00 AM" src="https://user-images.githubusercontent.com/21180050/164282130-8af84928-a4ca-45da-82e7-fc096054d5e8.png">